scenes/game/local_game.gd

```diff
@@ -43,7 +44,7 @@ const StrikeStaticConditions = [
 	"canceled_this_turn",
 ]
 
-var event_queue = []
+var event_queue = []  # This is purely a logging utility
```

Comment:
no it isn't. This is how the UI functions.

Reply:
Ah, I see my mistake. I was looking for code that reads `event_queue` but forgot that most code will access it through `get_latest_events()`.